tests/data/github/repos.py

```diff
@@ -103,52 +110,8 @@
             'login': 'example_org',
             '__typename': 'Organization',
         },
-        'collaborators': {
-            'edges': [
-                {'permission': 'WRITE'},
-                {'permission': 'WRITE'},
-                {'permission': 'WRITE'},
-                {'permission': 'WRITE'},
-                {'permission': 'WRITE'},
-            ],
-            'nodes': [
-                {
-                    'url': 'https://github.com/marco-lancini',
-                    'login': 'marco-lancini',
-                    'name': 'Marco Lancini',
-                    'email': 'm@example.com',
-                    'company': 'ExampleCo',
-                },
-                {
-                    'url': 'https://github.com/sachafaust',
-                    'login': 'sachafaust',
-                    'name': 'Sacha Faust',
-                    'email': 's@example.com',
-                    'company': 'ExampleCo',
-                },
-                {
-                    'url': 'https://github.com/SecPrez',
-                    'login': 'SecPrez',
-                    'name': 'SecPrez',
-                    'email': 'sec@example.com',
-                    'company': 'ExampleCo',
-                },
-                {
-                    'url': 'https://github.com/ramonpetgrave64',
-                    'login': 'ramonpetgrave64',
-                    'name': 'Ramon Petgrave',
-                    'email': 'r@example.com',
-                    'company': 'ExampleCo',
-                },
-                {
-                    'url': 'https://github.com/roshinis78',
-                    'login': 'roshinis78',
-                    'name': 'Roshini Saravanakumar',
-                    'email': 'ro@example.com',
-                    'company': 'ExampleCo',
-                },
-            ],
-        },
```

Comment:
I largely replicated this data as is below, tho I did change the permissions levels so that they vary instead of all being 'WRITE'